configs/sites/orion/packages.yaml

```diff
@@ -90,10 +90,11 @@ packages:
     externals:
     - spec: cpio@2.11
       prefix: /usr
-  curl:
-    externals:
-    - spec: curl@7.29.0+ldap
-      prefix: /usr
+  # Don't use OS curl, doesn't support secure transfers
+  #curl:
```

Comment:
Remove or comment out?

Reply:
At least the comment to not use the OS curl should remain, lines 94 onward can be deleted.

Reply:
Done